libtinyaes/include/tinyaes/aes_ccm_s.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Extended AES Counter with CBC-MAC, a.k.a. CCM* mode implementation
+ *
+ * Copyright 2020 Phoenix Systems
+ * Author: Gerard Świderski
+ *
+ * Remarks: implementation based on RFC 3610, IEEE 802.15.4
+ *
+ * %LICENSE%
+ */
+
+#ifndef AES_CCM_S_H
+#define AES_CCM_S_H
+
+#include <stdint.h>
+
+/*
+ * NOTICE: to use this operation mode libtinyaes should be compiled with
+ * `WITH_AES_CCM_S` defined. This makes libtinyaes compact in size.
+ */
+
+struct AES_ctx;
```

Comment:
Sorry for not spotting it in two previous review :)

Symbol in header - potential problems with -fno-common